packages/bridge-ui/src/components/Bridge/NFTBridgeComponents/ImportStep/ScannedImport.svelte

```diff
@@ -61,10 +60,6 @@
   } else {
     canProceed = false;
   }
-
-  onDestroy(() => {
-    $selectedNFTs = [];
-  });
```

Comment:
This component is only used in `packages/bridge-ui/src/components/Bridge/NFTBridgeComponents/ImportStep/ImportStep.svelte` and that component sets `$selectedNFTs = [];` when it is mounted.